scripts/delete-inactive-helm-releases.sh

```diff
@@ -4,6 +4,13 @@ kubernetesCluster=${2:cft-preview-00-aks}
 defaultInactiveDays=${3:-3}
 declare -A inactiveDaysOverride=(["civil"]=1) # Value should be less than defaultInactiveDays defined above
 
+declare -A repos
+
+repos["civil-service"]="civil-service"
+repos["civil-camunda"]="civil-camunda-bpmn-definition"
+repos["civil-ccd"]="civil-ccd-definition"
```

Comment:
As this is a generic script can we get the repo from the release instead of hardcoding?

Reply:
That's what I hoped at the beginning. Unfortunately the release doesn't match "exactly" the repository name, nor does it follow the same convention so I was left with an explicit mapping.

I share your concern on this aspect however and I'm wondering whether we should create a separate script for civil. Or, rather, one script that runs "normally" and then exceptions to the rule for situations where we need to do customisations. This way the shared script would remain common and individual teams would have more control over their own versions if necessary.

As a matter of fact, I was hoping to discuss this over a call with you and Praveen.

Reply:
After discussion, here's the highlights:
- Chris' idea is to attach the name of the repository to the helm release as a label (there's already other labels) so that we will just need to read the label from there.
- Chris' idea has been agreed, but we'll do it in stages. The above will represent the MVP. In the future we will add the labels and remove this customisation